service/src/main/resources/db/migration/V411__invoiced_fee_decision.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE invoiced_fee_decision (
+    invoice_id uuid REFERENCES invoice(id) ON DELETE CASCADE,
+    fee_decision_id uuid REFERENCES fee_decision(id) ON DELETE CASCADE
+);
+
+ALTER TABLE invoiced_fee_decision ADD PRIMARY KEY (invoice_id, fee_decision_id);
+CREATE INDEX fk$invoiced_fee_decision_invoice_id ON invoiced_fee_decision (invoice_id);
```

Comment:
This index seems unnecessary because the column is already included in the leftmost position of the primary key